it/src/test/java/com/google/cloud/teleport/it/bigquery/BigQueryResourceManagerUtilsTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.bigquery;
+
+import static com.google.cloud.teleport.it.bigquery.BigQueryResourceManagerUtils.checkValidTableId;
+import static org.junit.Assert.assertThrows;
+
+import java.util.Arrays;
+import org.junit.Test;
+
+/** Unit tests for {@link com.google.cloud.teleport.it.bigquery.BigQueryResourceManagerUtils}. */
+public class BigQueryResourceManagerUtilsTest {
```

Comment:
A few valid tests for `checkValidTableId` may be good, to make sure it works as intended (we are just validating error cases now).

Reply:
Fixed.